extensions/machine-learning/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "machine-learning",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.2.0",
```

Comment:
Why 0.2.1 and not 0.3.0? We typically do minor version updates for new versions of the extensions. (not really any specific reason - but that's the convention we've used so it would be nice to keep things consistent IMO)

Reply:
I already shipped 0.2.0 with some changes in insider and that's the version for June release. I was thinking since I only had one small change after that, I should just change the minor.

Reply:
(minor is the second number - you're updating the revision here)

Sounds good